bokehjs/src/coffee/models/selections/interaction_policy.ts

```diff
@@ -25,6 +25,20 @@ export abstract class SelectionPolicy extends Model {
       return !source.selected.is_empty()
     }
   }
+
+  do_inspection(hit_test_result: HitTestResult, source: ColumnarDataSource): boolean {
+    if (hit_test_result === null) {
```

Comment:
Unless you distinguish `undefined` from `null` in this code, it's preferred to use `==` and `!=` operators.